test/9HCY.tml

```diff
@@ -0,0 +1,9 @@
+=== Need document footer before directives
+--- from: @ingydotnet
+--- tags: error
```

Comment:
I think the tags `directive` and `footer` should be added here

Reply:
Fixed by 45376d5